Blocks/SS/test.cc

```diff
@@ -77,15 +77,19 @@ TEST(StreamSystem, TestHelperClassSmokeTest) {
 namespace stream_system_test {
 
 struct DemoEntryPublisherImpl {
+  using MutexLockStatus = current::locks::MutexLockStatus;
+
   size_t index = 0u;
   std::deque<DispatchDemoEntry> values;  // Use `deque` to avoid extra copies.
 
+  template <MutexLockStatus MLS>
```

Comment:
Unused template parameters can be unnames, `template <MutexLockStatus>`. Another nit: If a fully-qualified name fits the line, it's better for test readability and copy-and-paste-ability, ex. `template <current::locks::MutexLockStatus>`.